rpm/ofono.spec

```diff
@@ -130,6 +130,9 @@ systemctl daemon-reload ||:
 %postun
 systemctl daemon-reload ||:
 
+%transfiletriggerin -- %{_libdir}/ofono/plugins
+systemctl restart ofono.service ||:
```

Comment:
This should be actually try-restart, so that it only restarts it if it is already runnin, right?

Reply:
Fixed